pulsar-proxy/src/main/java/org/apache/pulsar/proxy/server/ProxyConfiguration.java

```diff
@@ -332,10 +332,10 @@
     private boolean forwardAuthorizationCredentials = false;
     @FieldContext(
         category = CATEGORY_AUTHENTICATION,
-        doc = "Whether the '/metrics' endpoint requires authentication. Defaults to true."
+        doc = "Whether the '/metrics' endpoint requires authentication. Defaults to false."
             + "'authenticationEnabled' must also be set for this to take effect."
     )
-    private boolean authenticateMetricsEndpoint = true;
+    private boolean authenticateMetricsEndpoint = false;
```

Comment:
This is a breaking change, related to security.

It is fine to make configuration more strict but not less strict  otherwise users who upgrade may not notice the change.

It is fine to be to default to requiring auth on the broker by default but not to change this default value in the proxy

Reply:
ok